c7n/mu.py

```diff
@@ -1071,8 +1071,8 @@ def render_event_pattern(self):
             payload['detail-type'] = events
         elif event_type == 'phd':
             payload['source'] = ['aws.health']
-            payload.setdefault('detail', {})
             if self.data.get('events'):
+                payload.setdefault('detail', {})
```

Comment:
this isn't quite correct, if you have categories set, they would bomb out with keyerror in the next if statement

Reply:
I realized on tests, pushed a change